lms/static/js/verify_student/pay_and_verify.js

```diff
@@ -55,6 +55,8 @@ var edx = edx || {};
                     function(isVisible) { return isVisible; }
                 ),
                 upgrade: el.data('msg-key') === 'upgrade',
+                currencyCode: el.data('default-currency-code'),
```

Comment:
@renevatium you want the currency code to appear uppercased consistently. You missed a few instances that would have caused the currency code to appear lowercased in a few spots. Doing it once here means you don't need to worry about it anywhere else.